build.sbt

```diff
@@ -13,22 +14,20 @@
  * =========================================================================================
  */
 
-val kamonCore         = "io.kamon"                  %%  "kamon-core"                  % "0.6.7"
-val kamonTestkit      = "io.kamon"                  %%  "kamon-testkit"               % "0.6.7"
-val newrelic          = "com.newrelic.agent.java"   %   "newrelic-agent"              % "3.26.1"
-val sprayJson         = "io.spray"                  %%  "spray-json"                  % "1.3.3"
-val sprayJsonLenses   = "net.virtual-void"          %%  "json-lenses"                 % "0.6.2"
-val scalaTest         = "org.scalatest"             %%  "scalatest"                   % "3.0.3"
-val scalaMock         = "org.scalamock"             %%  "scalamock-scalatest-support" % "3.6.0"
-val scalajHttp        = "org.scalaj"                %% "scalaj-http"                  % "2.3.0"
+name := "kamon-newrelic-reporter"
+version := "0.0.2-SNAPSHOT"
+description := "New Relic Kamon Reporter"
+
+scalaVersion := "2.13.0"
+crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.2")
```

Comment:
if 2.13 is the value of `scalaVersion`, does it also need to be in `crossScalaVersions`?

Reply:
It should be part of it, but we don't need to set it explicitly since it is already part of the sbt umbrella plugin. Don't worry about that, though. I will tweak the build if needed before releasing.